types/src/build_version/mod.rs

```diff
@@ -17,13 +21,13 @@ const MINOR_ONLY_MASK: u32 = 0x00ff00;
 const PATCH_ONLY_MASK: u32 = 0x0000ff;
 
 // api versions
-const FULL_API_VERSION: Version = new_version(1, 1, 0);
-const MINER_API_VERSION: Version = new_version(0, 15, 0);
-const WORKER_API_VERSION: Version = new_version(0, 15, 0);
+const FULL_API_VERSION: Version = Version::new(1, 1, 0);
+const MINER_API_VERSION: Version = Version::new(0, 15, 0);
+const WORKER_API_VERSION: Version = Version::new(0, 15, 0);
+
+const GIT_HASH: &str = git_version!(args = ["--always", "--exclude", "*"], fallback = "unknown");
 
 lazy_static! {
-    pub static ref CURRENT_COMMIT: String = git_version!(fallback = "unknown").to_string();
-    pub static ref BUILD_TYPE: RwLock<BuildType> = RwLock::new(BuildType::BuildDefault);
     pub static ref RUNNING_NODE_TYPE: RwLock<NodeType> = RwLock::new(NodeType::Full);
```

Comment:
Use `parking_lot::RwLock` and get rid of the `lazy_static`?

Reply:
Done!